src/python/traces/sampling/test/traces_test.go

```diff
@@ -0,0 +1,12 @@
+package test
+
+import (
+	"testing"
+
+	tu "github.com/joaopgrassi/otel-recipes/internal/common/testutils"
+)
+
+func TestSamplingSpans(t *testing.T) {
+	tc := tu.NewTraceTestCase("python.traces.sampling", "SamplingSpan", tu.StringAttribute("sampled", "1"), tu.StringAttribute("sampler", "traceidratio"))
```

Comment:
I wonder here if we should also assert that only 1 span was exported.. otherwise we don't really test that sampling is actually working, right? 

In that case you need to add the code directly here for asserting instead of using `tu.AssertSpanWithAttributeExists(t, tc)`. Would be better to have a more generic but we can start simple for now and improve later, either way is fine.

Reply:
Yes. In the example I'm generating 10k spans since if I generate only 1001 spans there's no guarantee that we will always have 1 span since python sdk makes a random sampling result based on the sampling probability given.

Reply:
The main thing with this test is we can only have sure about the number of sampled spans after run the python code. In SDks they test the sampler by just passing a static traceId which is in the range of the bound for the test. So it always works. For our case, we can't do that since the recipe should work exactly as is.